src/current/v24.3/ui-storage-dashboard.md

```diff
@@ -77,24 +89,46 @@ If you are running multiple nodes on a single machine (not recommended), the act
 
 For Windows systems, you can ignore the File Descriptors graph because the concept of file descriptors is not applicable to Windows.
 
+## Disk Write Breakdown
+
+- In the node view, the graph shows the number of bytes written to disk per second categorized according to the source for that node.
+
+- In the cluster view, the graph shows the number of bytes written to disk per second categorized according to the source for each node.
+
+Possible sources of writes with their series label are:
+
+- [WAL]({% link {{ page.version.version }}/architecture/storage-layer.md %}#memtable-and-write-ahead-log) (`pebble-wal`)
+- [Compactions]({% link {{ page.version.version }}/architecture/storage-layer.md %}#compaction) (`pebble-compaction`)
+- [SSTable ingestions]({% link {{ page.version.version }}/architecture/storage-layer.md %}#ssts) (`pebble-ingestion`)
+- [Memtable flushes]({% link {{ page.version.version }}/architecture/storage-layer.md %}#memtable-and-write-ahead-log) (`pebble-memtable-flush`)
+- [Raft snapshots]({% link {{ page.version.version }}/architecture/replication-layer.md %}#snapshots) (`raft-snapshot`)
+- [Encryption Registry]({% link {{ page.version.version }}/security-reference/encryption.md %}#encryption-keys-used-by-cockroachdb-self-hosted-clusters) (`encryption-registry`)
+- [Logs]({% link {{ page.version.version }}/logging-overview.md %}) (`crdb-log`)
+- SQL row spill (`sql-row-spill`), refer to [`cockroach start` command]({% link {{ page.version.version }}/cockroach-start.md %}#flags) flag `--max-disk-temp-storage`
+- [SQL columnar spill]({% link {{ page.version.version }}/vectorized-execution.md %}#disk-spilling-operations) (`sql-col-spill`)
+
+Refer to the **Hardware** dashboard [**Disk Write Bytes** graph]({% link {{ page.version.version }}/ui-hardware-dashboard.md %}#disk-write-bytes) to view an aggregate of all disk writes.
```

Comment:
```suggestion
To view an aggregate of all disk writes, refer to the **Hardware** dashboard [**Disk Write Bytes** graph]({% link {{ page.version.version }}/ui-hardware-dashboard.md %}#disk-write-bytes).
```

Reply:
I think pre-empting the link with the reason to follow the link is helpful for the reader. This is non-blocking, so take the suggestion if you want!